.gitignore

```diff
@@ -23,3 +23,4 @@
 
 # temporary files from the twisted.trial test runner
 _trial_temp/
+.tox/
```

Comment:
Can we keep this out of the .gitignore for now, unless it's causing problems. Once I'm done learning/testing lbryum I'll move onto lbry and will probably set it up the same way as lbryum (with codecove and tox with python2 and python3).

Tox makes it easy to test code under different versions of Python and also to make sure it's packaged properly (as a Python pip'able package). Which regular testing from source doesn't verify.

Reply:
removed, I'd added it because I was messing around with a tox configuration